src/main/features/claim/routes/claim-receipt.ts

```diff
@@ -0,0 +1,43 @@
+import * as express from 'express'
+import { Paths } from 'claim/paths'
+import { ErrorHandling } from 'common/errorHandling'
+import { DocumentsClient } from 'app/documents/documentsClient'
+import * as http from 'http'
+import * as HttpStatus from 'http-status-codes'
+import { Claim } from 'claims/models/claim'
+
+const documentsClient: DocumentsClient = new DocumentsClient()
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.receiptReceiver.uri,
+    ErrorHandling.apply(
+      async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+
+        const { externalId } = req.params
+        const claim: Claim = res.locals.user.claim
+
+        documentsClient.getClaimIssueCopy(externalId)
+          .on('response', (response: http.IncomingMessage) => {
+            if (response.statusCode !== 200) {
+              return next(new Error('Unexpected error during document retrieval'))
+            }
+            const buffers: Buffer[] = []
+            response.on('data', (chunk: Buffer) => {
+              buffers.push(chunk)
+            })
+            response.on('end', () => {
+              const pdf = Buffer.concat(buffers)
+              res.writeHead(HttpStatus.OK, {
+                'Content-Type': 'application/pdf',
+                'Content-Disposition': `attachment; filename=${claim.claimNumber}-Claim-Receipt.pdf`,
```

Comment:
What about `-claim-receipt.pdf` (lowercase)? Any reason to start with capitals? Requirement from @BenClancy ?

Reply:
we do lower case else where I believe

Reply:
I talked to Ben about it and he liked the look of it for now. He said he would have a think about it. Any update @BenClancy ?